2-mandatory/2-financial-times.js

```diff
@@ -5,7 +5,10 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    const smallArticle = allArticleTitles.filter(article => {
```

Comment:
if it's a one liner feel free to immediately `return` the value without assigning it to a variable